src/NodeAnalyzer/LaravelViewFunctionMatcher.php

```diff
@@ -32,26 +36,93 @@ public function __construct(
     /**
      * @return RenderTemplateWithParameters[]
      */
-    public function match(FuncCall|StaticCall $funcCall, Scope $scope): array
+    public function match(CallLike|ClassMethod $funcCall, Scope $scope): array
     {
-        $funcName = $funcCall->name;
+        if ($funcCall instanceof FuncCall
+            && $funcCall->name instanceof Name
+            && $scope->resolveName($funcCall->name) === 'view'
+        ) {
+            return $this->matchView($funcCall, $scope);
+        }
 
-        if ($funcCall instanceof FuncCall) {
-            if (! $funcName instanceof Name
-                || $scope->resolveName($funcName) !== 'view') {
-                return [];
-            }
+        if ($funcCall instanceof StaticCall
+            && $funcCall->class instanceof Name
+            && (string) $funcCall->class === View::class
+            && $funcCall->name instanceof Identifier
+            && (string) $funcCall->name === 'make'
+        ) {
+            return $this->matchView($funcCall, $scope);
+        }
+
+        if ($funcCall instanceof ClassMethod
+            && (string) $funcCall->name === 'content'
+        ) {
+            return $this->matchContent($funcCall, $scope);
+        }
+
+        return [];
+    }
+
+    /**
+     * @return RenderTemplateWithParameters[]
+     */
+    private function matchContent(ClassMethod $funcCall, Scope $scope): array
```

Comment:
This is way to specific, it won't work if `Content` is being constructed in a sub method or passed as a property. I'll take a quick look if I can resolve it.